### PR TITLE
[zh] synchronize translate components.md

### PR DESCRIPTION
**Pull request by shuaijinchao**
/language zh

### CODE REVIEW

File: content/zh/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -29,19 +29,16 @@ When you deploy Kubernetes, you get a cluster.
...
 <!-- overview -->
 当你部署完 Kubernetes, 即拥有了一个完整的集群。
 {{< glossary_definition term_id="cluster" length="all" prepend="一个 Kubernetes">}}
 
 本文档概述了交付正常运行的 Kubernetes 集群所需的各种组件。
 
-这张图表展示了包含所有相互关联组件的 Kubernetes 集群。
-
-![Kubernetes 组件](/images/docs/components-of-kubernetes.svg)
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="Kubernetes的组件" caption="Kubernetes集群的组件" class="diagram-large" >}}
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/components.md:41]:
```suggestion
{{< figure src="/images/docs/components-of-kubernetes.svg" alt="Kubernetes 的组件" caption="Kubernetes 集群的组件" class="diagram-large" >}}
```

Comment (shuaijinchao) [reply content/zh/docs/concepts/overview/components.md]: done

**Pull request merged**